stdlib/array.gr

```diff
@@ -880,6 +972,9 @@ let wrapNegativeIndex = (arrLen, idx) => {
  * @param endIndex: The index of the array where the slice will end (exclusive)
  * @param array: The array to be sliced
  * @returns The subset of the array that was sliced
+ * 
+ * @example
+ * Array.slice(1, 3, [> 5, 6, 7, 8, 9]) // [> 6, 7]
```

Comment:
Same As Above:
```suggestion
 * Array.slice(1, 3, [> 5, 6, 7, 8, 9]) == [> 6, 7]
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....